modules/flowable-app-engine-rest/src/test/java/org/flowable/app/rest/service/api/repository/AppDefinitionResourceTest.java

```diff
@@ -86,8 +86,9 @@ public void testGetAppDefinitionResourceData() throws Exception {
         // Check "OK" status
         String content = IOUtils.toString(response.getEntity().getContent(), StandardCharsets.UTF_8);
         closeResponse(response);
-        assertThat(content).isNotNull();
-        assertThat(content.contains("oneApp")).isTrue();
+        assertThat(content)
+                .isNotNull()
```

Comment:
In my opinion it would be even more idiomatic to remove the `isNotNull` check, because AssertJ already handle that.

Reply:
I went back and forth on that; there was a lot of code written in that style and I didn't feel good about removing it all.  Maybe next time :smile:

Reply:
😄